packaging/python/build_wheels.bash

```diff
@@ -28,29 +28,42 @@ py_ver=""
 setup_venv() {
     local py_bin="$1"
     py_ver=$("$py_bin" -c "import sys; print('%d%d' % tuple(sys.version_info)[:2])")
-    local venv_dir="nrn_build_venv$py_ver"
+    suffix=$("$py_bin" -c "print(str(hash(\"$py_bin\"))[0:8])")
+    local venv_dir="nrn_build_venv${py_ver}_${suffix}"
 
-    if [ "$py_ver" -lt 35 ]; then
+    if [ "$py_ver" -lt 35 ] &&  ["$py_ver" -ge 30 ]; then
```

Comment:
The condition doesnt include `-lt 2.7` maybe in a follow up PR we can change this to `-ne 2.7` (so using 2.7 as the only exception) which can be dropped later.